storage/rocksdb/build_rocksdb.cmake

```diff
@@ -146,14 +147,41 @@ int main() {
   endif()
 endif()
 
+CHECK_CXX_SOURCE_COMPILES("
```

Comment:
any reason `CHECK_FUNCTION_EXISTS` wasn't used here?

Reply:
From what I can tell, this verifies the presence of the `SYNC_FILE_RANGE_WRITE` flag in the function call.

Reply:
`CHECK_SYMBOL_EXIST(SYNC_FILE_RANGE_WRITE "fcntl.h" HAVE_SYNC_FILE_RANGE_WRITE)`